lightkurve/tests/test_periodogram.py

```diff
@@ -390,3 +395,9 @@ def test_bls_period():
     with pytest.raises(ValueError) as err:  # NaNs should raise a nice error message
         lc.to_periodogram(method="bls", period=[1, 2, 3, np.nan, 4])
     assert("period" in err.value.args[0])
+
+
+def test_waterfall():
+    lc = LightCurve(time=np.arange(100), flux=np.random.normal(1, 0.01, 100))
+    ax = lc.fold(10, 1).waterfall()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'ax' is assigned to but never used](https://app.codacy.com/manual/barentsen/lightkurve/pullRequest?prid=4938792)